assets/src/edit-story/elements/image/scalePanel.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import InOverlay from '../../components/overlay';
+
+const MIN_WIDTH = 165;
+const HEIGHT = 28;
+const OFFSET_Y = 8;
+// @todo: Should maxScale depend on the maximum resolution? Or should that
+// be left up to the helper errors? Both? In either case there'd be maximum
+// bounding scale.
+const MAX_SCALE = 400;
+
+const Container = styled.div`
+  position: absolute;
+  left: ${ ( { x, width } ) => `${ x + ( ( width - Math.max( width, MIN_WIDTH ) ) / 2 ) }px` };
+  top: ${ ( { y, height } ) => `${ y + height + OFFSET_Y }px` };
+  width: ${ ( { width } ) => `${ Math.max( width, MIN_WIDTH ) }px` };
+  height: ${ HEIGHT }px;
+
+  background: ${ ( { theme } ) => theme.colors.bg.v6 };
+  border-radius: 4px;
+
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  align-items: center;
+  padding: 0 4px;
+`;
+
+const Range = styled.input`
+  flex: 1 1;
+  margin: 4px;
+  min-width: 100px;
+  cursor: pointer;
+`;
+
+const ResetButton = styled.button`
```

Comment:
Is there no existing "standard" button for this? Is it a new type of standard button to be used elsewhere?

Reply:
Not sure if there's much more here than `<button>Reset</button>` at this time. The main "commons" candidate is `text-transform: uppercase`. But we might want to eventually apply to most of the editor buttons. All other styles are, for now, very tightly applied to the the "scale" panel and I've not yet seen any panel similar to it in designs.